docs/source/example_notebooks/graph_conditional_independence_refuter.ipynb

```diff
@@ -49,15 +49,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data = dowhy.datasets.linear_dataset(beta=10,\n",
-    "        num_common_causes=5,\n",
-    "        num_instruments = 2,\n",
-    "        num_effect_modifiers=1,\n",
-    "        num_samples=5000, \n",
-    "        treatment_is_binary=True,\n",
-    "        stddev_treatment_noise=10,\n",
-    "        num_discrete_common_causes=1)\n",
-    "\n",
+    "data = dowhy.datasets.generate_dataset(num_vars = 10,\n",
+    "                                       num_samples = 5000)\n",
```

Comment:
mention what the generate_dataset function does. Can you show which variables are discrete and which ones are continuous?